library/core/src/char/methods.rs

```diff
@@ -517,8 +518,9 @@ impl char {
     /// assert_eq!('"'.escape_default().to_string(), "\\\"");
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[rustc_const_stable(feature = "const_char_escape_default", since = "1.49.0")]
```

Comment:
This should be 1.50, not 1.49. 1.49 is currently in beta, new PRs land on nightly, which is 1.50. https://forge.rust-lang.org/

Reply:
Ah I see, I'll change it.